extensions/wikia/ArticleVideo/bower_components/html5-skin/js/components/thumbnailCarousel.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * ThumbnailCarousel component
+ *
+ * @module ThumbnailCarousel
+ */
+
+var React = require('react'),
+    ReactDOM = require('react-dom'),
+    Utils = require('./utils');
+
+var ThumbnailCarousel = React.createClass({
+  getInitialState: function() {
+    return {
+      thumbnailWidth: 0,
+      thumbnailHeight: 0,
+      centerThumbnailWidth: 0,
+      centerThumbnailHeight: 0,
+      thumbnailPadding: 6
+    };
+  },
+
+  componentDidMount: function() {
+    var thumbnail = ReactDOM.findDOMNode(this.refs.thumbnail);
+    var carousel = ReactDOM.findDOMNode(this.refs.thumbnailCarousel);
+    var thumbnailStylePadding = thumbnail ? window.getComputedStyle(thumbnail, null).getPropertyValue("padding") : 0;
+    thumbnailStylePadding = parseFloat(thumbnailStylePadding); // convert css px to number
+    var thumbnailPadding = !isNaN(thumbnailStylePadding) ? thumbnailStylePadding : this.state.thumbnailPadding;
+
+    if (thumbnail && carousel) {
+      if (thumbnail.clientWidth && carousel.clientWidth) {
+        this.setState({
+          thumbnailWidth: thumbnail.clientWidth,
+          thumbnailHeight: thumbnail.clientHeight,
+          centerThumbnailWidth: carousel.clientWidth,
+          centerThumbnailHeight: carousel.clientHeight,
+          thumbnailPadding: thumbnailPadding
+        });
+      } else {
+        var thumbnailStyleWidth = thumbnail ? window.getComputedStyle(thumbnail, null).getPropertyValue("width") : 0;
+        thumbnailStyleWidth = parseFloat(thumbnailStyleWidth); // convert css px to number
+        var thumbnailWidth = !isNaN(thumbnailStyleWidth) ? thumbnailStyleWidth : parseInt(this.props.thumbnailWidth);
+
+        var thumbnailStyleHeight = thumbnail ? window.getComputedStyle(thumbnail, null).getPropertyValue("height") : 0;
+        thumbnailStyleHeight = parseFloat(thumbnailStyleHeight); // convert css px to number
+        var thumbnailHeight = !isNaN(thumbnailStyleHeight) ? thumbnailStyleHeight : parseInt(this.props.thumbnailHeight);
+
+        var carouselStyleWidth = carousel ? window.getComputedStyle(carousel, null).getPropertyValue("width") : 0;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change this condition so that it does not always evaluate to "true"; some subsequent code is never executed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2583)